community/blog/2021-05 Enhanced request tracking in Arcus.WebApi v1.3/README.md

```diff
@@ -0,0 +1,135 @@
+# Enhanced request tracking in Arcus.WebApi v1.3
+
+The new version v1.3 of the [Arcus Web API](https://github.com/arcus-azure/arcus.webapi) project is all about request tracking. Previous versions introduced the request tracking concept as a way to track the incoming HTTP request and outgoing HTTP response with additional configurations on what should be included in the tracking information. This includes the request/response body, headers, the time it took to process the request...
```

Comment:
Maybe reduce a bit since the focus is on the newer version?
```suggestion
[Arcus Web API](https://github.com/arcus-azure/arcus.webapi) v1.3 is all about request tracking. Previous versions introduced the request tracking concept as a way to track the incoming HTTP request and outgoing HTTP response with a limited set of knobs to tweak such as to request/response body, headers, the time it took to process the request...
```